src/components/Comments.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { getComments } from "../utils";
+
+const Comments = ({article_id}) => {
+      const [comments, setComments] = useState([])
+      const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        const fetchComments = async () => {
+            const response = await getComments(article_id)
+            console.log(response.comments)
+            setComments(response.comments)
+            setIsLoading(false)
+        }
+        fetchComments()
+    }, [])
+
+    if (isLoading === true){
+        return <p>Loading...</p>
+    }
+    else return (
```

Comment:
If there are no comments, how does the user know? Would be good if we could have something rendered to the user to idnicate this is the case.